pkgs/development/compilers/mono/generic-cmake.nix

```diff
@@ -76,6 +76,10 @@ stdenv.mkDerivation rec {
   # [1] https://github.com/mono/mono/blob/master/scripts/gmcs.in
   + ''
     ln -s $out/bin/mcs $out/bin/gmcs
+  ''
```

Comment:
From the previous pr:

> Update on this: I cannot recall, but I believe (knowing me probably not) the upstream Mono maintainers told me to apply this. I completely forgot about this until yesterday, though, you might have to check.

I think we'd need a definitive answer on whether this is needed to merge this.

Reply:
I don't have a good build machine. Could you see if it still runs without it?